composer.json

```diff
@@ -40,7 +41,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "google/gax": "^1.19.1"
+        "google/gax": "dev-support-operations-for-compute-v2 as 1.34"
```

Comment:
Just leaving a comment to update this once release is cut.

Reply:
```suggestion
        "google/gax": "^1.34"
```